cmd/check_goversion/main.go

```diff
@@ -22,7 +23,8 @@ func main() {
 	p.StringFlag("endpoint", "", "The page to check for your application's Go version. The page must output JSON.")
 	p.StringFlag("endpoint-path", "", "The path to the JSON key containing your application's Go version specified in JavaScript notation")
 	p.StringFlag("version", "latest", "The expected version. If set to `latest`, the latest-version-url will be consulted")
-	p.StringFlag("latest-version-url", "https://golang.org/VERSION?m=text", "The url where one can retrieve the latest version of Go")
+	p.StringFlag("latest-version-filter", "go1.10", "A prefix filter for the version.  Example `go1.10`")
```

Comment:
what if we removed this filter and just said that any version marked stable on that download URL is ok?

Reply:
Sure thing.  I can do that. :)